piaxe/miner.py

```diff
@@ -391,6 +393,7 @@ def __init__(self, config, address, network):
 
         self.shares = list()
         self.stats = influx.Stats()
+        self.display = SSD1306(self.stats)
```

Comment:
this shouldn't be called if display is not enabled - maybe move it into the `if` that creates the thread? :thinking: